src/zzz_od/gui/view/devtools/operation_debug_interface.py

```
@@ -130,5 +127,13 @@ def _on_del_clicked(self) -> None:
 
         self._update_auto_battle_config_opts()
 
-    def _on_gamepad_type_changed(self, idx: int, value: str) -> None:
-        self.ctx.battle_assistant_config.control_method = value
+    def _on_background_mode_changed(self, value: bool) -> None:
+        if value and not pc_button_utils.is_vgamepad_installed():
+            self.background_mode_opt.setValue(False, emit_signal=False)
+            self.ctx.battle_assistant_config.background_mode = False
+            InfoBar.warning(
+                title='后台模式不可用',
+                content='未检测到 vgamepad / ViGEmBus，请先安装虚拟手柄驱动',
```

Ruff (0.15.20)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)